### PR TITLE
Add eth-bridge prometheus metrics

### PR DESCRIPTION
**Pull request by vovac12**
Added metrics:
* Substrate network height of the next block to be processed
* Ethereum network height of the next block to be processed
* Ethereum network height of the latest received block
* Pending transactions count
* Failed pending transactions count

### CODE REVIEW

File: misc/eth.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+    "networks": {
+        "0": {
+            "url": "http://127.0.0.1:8545"
+        }
+    }
+}
```
Comment (wer1st) [inline misc/eth.json:7]: Add empty line

File: node/src/eth_bridge_metrics.rs [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+            ethereum_height,
+            period,
+            backend,
+        })
+    }
+
+    pub fn get_offchain_value<T>(
+        storage: &<B as sc_client_api::Backend<Block>>::OffchainStorage,
+        key: &[u8],
+        desciption: &str,
```
Comment (Alexey-N-Chernyshov) [inline node/src/eth_bridge_metrics.rs:94]: typo: `desciption` -> `description`

**Review by wer1st — APPROVED** [review]

**Review by pakrentos — APPROVED** [review]

File: node/src/eth_bridge_metrics.rs [hunk 3]
```diff
@@ -0,0 +1,175 @@
...
+    STORAGE_FAILED_PENDING_TRANSACTIONS_KEY, STORAGE_NETWORK_IDS_KEY,
+    STORAGE_PENDING_TRANSACTIONS_KEY, STORAGE_SUB_TO_HANDLE_FROM_HEIGHT_KEY,
+};
+use framenode_runtime::Runtime;
+use framenode_runtime::{eth_bridge::offchain::SignedTransactionData, opaque::Block};
+use prometheus_endpoint::{register, Gauge, Opts, PrometheusError, Registry, U64};
+use sp_core::H256;
+use sp_runtime::offchain::OffchainStorage;
+use std::collections::BTreeSet;
+use std::{collections::BTreeMap, sync::Arc};
```
Comment (Alexey-N-Chernyshov) [inline node/src/eth_bridge_metrics.rs:12]:
```suggestion
use std::collections::{BTreeSet, BTreeMap};
use std::sync::Arc;
```

**Review by Alexey-N-Chernyshov — APPROVED** [review]

File: node/src/eth_bridge_metrics.rs [hunk 4]
```diff
@@ -0,0 +1,174 @@
+use codec::Decode;
```
Comment (Alexey-N-Chernyshov) [inline node/src/eth_bridge_metrics.rs:1]: Just realized that copyright is missing.

**Pull request merged**